inst/include/plant/resource_spline.h

```diff
@@ -0,0 +1,136 @@
+// -*-c++-*-
+
+#ifndef PLANT_PLANT_RESOURCE_SPLINE_H_
+#define PLANT_PLANT_RESOURCE_SPLINE_H_
+
+#include <plant/interpolator.h>
+#include <plant/adaptive_interpolator.h>
+#include <plant/ode_interface.h>
+#include <plant/util.h>
+
+using namespace Rcpp;
+
+namespace plant {
+
+class Resource_spline {
```

Comment:
Minor nitpick, but I think classes have mostly used CamelCase like StochasticPatch and AdaptiveInterpolator. We do use underscores for *_Strategy and *_Environment, but I think this would be best as ResourceSpline.

Reply:
Agree